docs/source/changelog/experimental.md

```diff
@@ -0,0 +1,89 @@
+# Migrating from v0.19.0 to v0.20.0
+
+This constitutes a list of all the changes needed to migrate your code
+to work with the latest version of Manim
+
+## Manager
+If you ever used `Scene.render`, you must replace it with {class}`.Manager`:
+```py
+manager = Manager(SceneClass)
+manager.render()
+```
+
+If you are a plugin author that overrode `Scene.render`, you should migrate
+your code to use the specific public methods on {class}`.Manager` instead.
+
+## ThreeDScene and Camera
+`ThreeDScene` has been completely removed, and all of its functionality has been replaced
+with methods on {class}`.Camera`, which can be accessed via {attr}`.Scene.camera`.
+
+For example, the following code
+```py
+class MyScene(ThreeDScene):
+    def construct(self):
+        t = Text("Hello")
+        self.add_fixed_in_frame_mobjects(t)
+        self.begin_ambient_camera_rotation()
+        self.wait(3)
+```
+Should be changed to
+```py
+# change ThreeDScene -> Scene
+class MyScene(Scene):
+    def construct(self):
+        t = Text("Hello")
+        # add_fixed_in_frame_mobjects no longer exists,
+        # have to do it manually
+        t.fix_in_frame()
+        self.add(t)
+
+        # access the method on the camera
+        self.camera.begin_ambient_rotation()
```

Comment:
The original `ThreeDScene` method added the camera to the scene. Now we have to add the camera manually:

```suggestion
	# Access the method on the camera.
        self.camera.begin_ambient_rotation()
        self.add(self.camera)
```